pulp_ansible/app/viewsets.py

```diff
@@ -234,19 +332,88 @@ class CollectionDeprecatedViewSet(ContentViewSet):
     endpoint_name = "collection_deprecations"
     queryset = AnsibleCollectionDeprecated.objects.all()
     serializer_class = CollectionSerializer
+    queryset_filtering_required_permission = "ansible.view_collection"
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": "update",
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_model_or_obj_perms:ansible.change_collection",
+                    "has_model_or_obj_perms:ansible.view_collection",
+                    "has_model_or_obj_perms:ansible.change_ansiblerepository",
```

Comment:
Change action to `create` and change the condition to match the first two conditions of CollectionVersion create.